.github/workflows/ci-model-regression.yml

```
@@ -596,6 +596,11 @@ jobs:
         if: steps.set_dataset_config_vars.outputs.is_dataset_exists == 'true' && steps.set_dataset_config_vars.outputs.is_config_exists == 'true'
         run: poetry run pip install -U datadog-api-client ddtrace
 
+      - name: Validate that GPUs are not working
```

```suggestion
      - name: CPU run: Validate that no GPUs are available
```
Nitpicking here, but something about GPUs not working seems confusing 😅